src/Arcus.WebApi.Logging.Core/Correlation/HttpCorrelation.cs

```diff
@@ -361,7 +361,7 @@ private void AddCorrelationResponseHeaders(HttpContext httpContext, string reque
                 {
                     if (string.IsNullOrWhiteSpace(requestId))
                     {
-                        _logger.LogWarning("No response header was added given no operation parent ID was found");
+                        _logger.LogTrace("No response header was added given no operation parent ID was found");
```

Comment:
Am I missing something ?  The related issue talks about another log-entry that should have a 'trace' level, but it is not included in the PR ?

Reply:
The issue was still using v1.6, while this is already fixed in the preview version of v1.6.1-preview-1; which is also in the issue discussion:

> Upgraded to version v1.6.1-preview-1 and did a test, the Operation correlation ID's cannot be added to response, but the Operation.IncludeInResponse setting was still set trace messages has indeed disappeared.